components/App.jsx

```diff
@@ -47,14 +49,15 @@ const App = () => {
                 <GlobalStyle />
                 <AudioButton />
 
-                {isNav && <MainNav setIsNav={setIsNav} setIsSpeechBoxActive={setIsSpeechBoxActive} />}
+                {isNav && <MainNav setIsNav={setIsNav} isNav={isNav} setIsSpeechBoxActive={setIsSpeechBoxActive} />}
```

Comment:
I might rename this to `isNavActive`. I'm not sure what `isNav` really says here.